src/pages/animation.ts

```diff
@@ -271,3 +272,43 @@ function initialConfiguration(
 export function addAnimationListener() {
   setGradientDegreeValue(getDegreeElement);
 }
+
+
+
+function resetValues() {
+  const { inputs } = getAllFields(attribute);
+  const resetBtn = document.querySelector("[data-reset='animation']") as HTMLButtonElement;
```

Comment:
Since we use this every time, just add it to the get Elements file as a function, the function will receive an argument for which generator it's in